src/components/Staking/IdoStakingForm.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Box, Button, Heading, Image, Input, InputGroup, Spacer, Stack, Switch, Text
+} from '@chakra-ui/react'
+import { observer } from 'mobx-react-lite'
+import { Contract } from 'near-api-js/lib/contract'
+import { useStakingForm_Stake } from './StakingHooks'
+import StakingStatsStore from './StakingStatsStore'
+
+const darkPurple200 = '#2D225E'
```

Comment:
if its a common color. could u export these color from Figma and copy to "src/styles/main.css" like:
```
:root {
  --xxxx1: #1e90ff;
  --xxxx2: #1e90ff;
....
}
```
then we can use var css syntax. But right now all is ok. we can do it later

Reply:
Because we not finish UI yet and colors are only used in this component. 

Also need to choose chakra or tailwindcss to make it easier to code in the future.